addons/mail/static/src/models/partner/partner.js

```diff
@@ -307,6 +315,9 @@ function factory(dependencies) {
         messagesAsAuthor: one2many('mail.message', {
             inverse: 'author',
         }),
+        messaging: many2one('mail.messaging', {
+            compute: '_computeMessaging',
+        }),
```

Comment:
Add this in `hr_holidays` with the rest, this is only used in `hr_holidays` and serves only as compute dependency (add the usual comment).

Reply:
in fact I put it there as in master it will be added there

Reply:
ok doesn't matter too much